code/SiteConfig.php

```diff
@@ -78,6 +78,8 @@ class SiteConfig extends DataObject implements PermissionProvider, TemplateGloba
         'CMS_ACCESS_LeftAndMain'
     ];
 
+    private static bool $require_sudo_mode = true;
```

Comment:
When we talked about this, we agreed that this would only be for the "access" tab. There's already permissions controlling access to this model as a whole.

Reply:
My memory of this was "try and get it to work for just a single tab, and if that proves to be too difficult just put it on everything", and it quickly proved to be too difficult.  I'm assuming that editing siteconfig is very infrequent operation so the UX downside here should be very low

The permissions controlling access to the model is irrelevant, the whole point of this is to prevent XSS style attacks on users who do have access

Reply:
Putting the discussion about tab vs model aside for now, if we're not protecting the access controls in `SiteTree` and `File`, we shouldn't protect them here.

It's sending mixed messages - "we think access controls are privilege escalation" in siteconfig but "we think access controls are _not_ privilege escalation" in the individual records, when they control the same thing.

Reply:
Fair enough. Will close PR.